src/RepairManager/test/test_ecc_detect_error_rule.py

```diff
@@ -107,54 +44,14 @@ def _mock_prometheus_error_data():
                     }]
                 }
         }
-    return mock_prometheus_error_data
+    return mock_prometheus_latency_data
 
 class Testing(unittest.TestCase):
```

Comment:
Usually the test class for a class or a file is named `Test<CamelCaseClassOrFileName>`, hence `TestEccDetectErrorRule`